src/airport.js

```diff
@@ -36,16 +38,6 @@ const airport = {
     {
         return this.landedPlanes.splice(planes, 1);
     },
-
```

Comment:
Glad I decided to not use an extra function, this helped mitigate errors/failing tests and made the code overall cleaner.